Tests/UnitTests/Purchasing/Purchases/PurchasesPurchasingTests.swift

```diff
@@ -98,7 +98,7 @@ class PurchasesPurchasingTests: BasePurchasesTests {
     }
 
     func testFinishesTransactionsIfSentToBackendCorrectly() throws {
-        var finished = true
+        var finished = false
```

Comment:
This test was probably not doing anything before :)

Reply:
🤦